airflow/api_connexion/schemas/dag_run_schema.py

```diff
@@ -74,6 +74,7 @@ class Meta:
     last_scheduling_decision = auto_field(dump_only=True)
     run_type = auto_field(dump_only=True)
     note = auto_field(dump_only=False)
+    triggered_by = auto_field(dump_only=True)
```

Comment:
Not sure how this field would work if the enum does not subclass from str. We’ll find out with tests.

Reply:
It is not working with this way, I changed the line with this:
```python
    triggered_by = fields.Enum(DagRunTriggeredByType, by_value=True, dump_only=True)
```
I am trying to run the tests and see everything is fine or not